src/components/Icon/CrownIcon/CrownIcon.spec.js

```diff
@@ -0,0 +1,8 @@
+import { icons, common } from '../../../util/generic-tests';
+
+import CrownIcon from './CrownIcon';
+
+describe('CrownIcon', () => {
+	common(CrownIcon);
```

Comment:
I think there should be a snapshot generated from this

Reply:
updated